README.md

```diff
@@ -1,124 +1,32 @@
-# Firmware/Certificates updater for WINC and NINA Wifi module
+# arduino-fwuploader
```

Comment:
What is the product name? I see it is referred to as "Arduino Firmware Uploader" below. I recommend using the product name here.

Note that the product name and the command name can be different:
https://arduino.atlassian.net/wiki/spaces/ATL/pages/2131034179/Naming+conventions#Product-Name

Reply:
Seems legit. I'll use "Arduino Firmware Uploader" then